examples/gitlab-ha/vars.mk.sample

```diff
@@ -2,13 +2,8 @@ export DNS_ZONE_NAME=example.net
 export ENVIRONMENT_NAME=my-test
 export GITLAB_NAME=gitlab
 export REGION=us-east-1
-export REGISTRY_NAME=registry
-export CA_NAME=Example Net CA
-export ORGANIZATION=Example Net
-export COUNTRY=US
-export STATE=CA
-export LOCALITY=SF
+export GITLAB_REGISTRY_NAME=registry
 export GITLAB_URL=${GITLAB_NAME}.${DNS_ZONE_NAME}
-export REGISTRY_URL=${REGISTRY_NAME}.${DNS_ZONE_NAME}
+export REGISTRY_URL=${GITLAB_REGISTRY_NAME}.${DNS_ZONE_NAME}
 export REGISTRY_S3_BUCKET_NAME=abc123def456
```

Comment:
is this not used and cruft from the past?

Reply:
its actually a buglet and causes problems because it's named this way in the module. I fixed it in the simple-ha and didn't notice the problem here, so took the opportunity to fix it now.